test/built-ins/TypedArray/prototype/toReversed/length-property-ignored.js

```diff
@@ -37,9 +39,10 @@ testWithTypedArrayConstructors(TA => {
 
   length = 2;
   var res = ta.toReversed();
-  assert.compareArray([res[0], res[1], res[2], res[3]], [2, 1, 0, undefined]);
+  assert.compareArray(res, [2, 1]);
+  assertEq(res[2], 0);
```

Comment:
I'm only realizing now that `compareArray` is going to get confused from the overridden `length`. I'm not sure if I still agree with my prior suggestion here.

Alternatively, we could structure the test as:

For each constructor *TA*:
1. define property on prototype with value 2
2. call `toReversed`
3. delete property
4. inspect result
5. define property on prototype with value 5
6. call `toReversed`
7. delete property
8. inspect result

What do you think?

Reply:
That's fine -- I changed it a little bit to set the property to 3 instead of deleting the property, since if we delete the length, then it defaults to 0.